entrypoints/puppet-parser

```diff
@@ -13,16 +13,13 @@ if [[ ! -d "$DIR" ]]; then DIR="$PWD"; fi
 # shellcheck source=../shared/entrypoint-anubis-pre disable=SC1091
 . "$DIR/../shared/entrypoint-anubis-pre"
 
-# TODO: these hardcoded values will be replaced by output from something like
-# eval $(pdk environment --puppet-version=\${\$puppet_version})
-# Once that change is incorporated, we may want to consolidate the output generation and
-# error handling into a shared script.
-export PATH=$PATH:/opt/puppetlabs/pdk/private/ruby/2.5.8/bin:/opt/puppetlabs/pdk/private/ruby/2.5.8/lib/ruby/gems/2.5.0
-export GEM_PATH=/opt/puppetlabs/pdk/private/ruby/2.5.8/lib/ruby/gems/2.5.0:/opt/puppetlabs/pdk/share/cache/ruby/2.5.0:/opt/puppetlabs/pdk/private/puppet/ruby/2.5.0
+# TODO: consolidate the output generation and error handling into a shared script for use across all evaluations.
+version=${2:-7}
+eval $(pdk env --puppet-version=${version})
 
 # Run parser and emit output to json document
 {
-  /opt/puppetlabs/pdk/private/puppet/ruby/2.5.0/bin/puppet parser validate --render-as json manifests > parser_output.json
+  puppet parser validate --render-as json manifests > parser_output.json
```

Comment:
Did we need to include the `_version_` thing to activate the correct gem version on the path?

Reply:
The version is used in the `pdk env` command above, is that what you mean? (I realized I split this up into a couple commits, and the first of which _just_ added `pdk env` without the `--puppet-version` option, so that may have added to confusion.)

Reply:
I think we figured out that that only activates the right Ruby version for `pdk env` invocations though, since multiple puppet gems are installed into each Ruby version, I think we still need a way (since we aren't using bundler here) to activate the correct puppet gem when you call `puppet`.